exonum/src/node/mod.rs

```diff
@@ -948,8 +945,7 @@ impl Node {
     /// Creates node for the given services and node configuration.
     pub fn new(
         database: impl Into<Arc<dyn Database>>,
-        external_runtimes: impl IntoIterator<Item = impl Into<RuntimeInstance>>,
-        services: impl IntoIterator<Item = Box<dyn ServiceFactory>>,
+        with_runtimes: impl FnOnce(mpsc::Sender<UpdateEndpoints>) -> Vec<RuntimeInstance>,
```

Comment:
It makes sense to add a verbose explanation what does this argument mean and what should if contain.

Reply:
Does it make sense to create type alias for `FnOnce(mpsc::Sender<UpdateEndpoints>) -> Vec<RuntimeInstance>` ?

Reply:
I think that it is too verbose.